src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -202,6 +150,33 @@ def main(self):
             default=[],
             help='Devices to provision OSDs',
         )
+        parser.add_argument(
+            '--db-devices',
+            nargs='*',
+            type=arg_validators.ValidDevice(),
+            default=[],
+            help='Devices to provision OSDs db volumes',
+        )
+        parser.add_argument(
+            '--wal-devices',
+            nargs='*',
+            type=arg_validators.ValidDevice(),
+            default=[],
+            help='Devices to provision OSDs wal volumes',
+        )
+        parser.add_argument(
+            '--journal-devices',
+            nargs='*',
+            type=arg_validators.ValidDevice(),
+            default=[],
+            help='Devices to provision OSDs journal volumes',
+        )
+        parser.add_argument(
+            '--auto',
```

Comment:
why would one need to define `--auto` here instead of assuming automatic behavior if db/wal/journal aren't specified?

Reply:
Because without this it would be impossible to deploy standalone OSDs on a mixed group of drives (i.e. a bunch of spinners and ssd's).

Though to be fair `--no-auto` is probably the better choice here, will update.